src/frontend/csrs-portal/src/app/components/landing/landing.component.html

```diff
@@ -63,169 +51,196 @@ <h1 class="pt-2">Child Support Recalculation Service</h1>
             </a>
           </p>
           </div>
+        </div>
       </section>
     </div>
     <div class="container">
-    <section>
-      <div class="flex-6-col round3">
-        <mat-grid-list cols="6" rowHeight="60px" gutterSize="5px">
-          <mat-grid-tile colspan="1" rowspan="2" aria-rowindex="0" class="mi-tile-contents-center">
+    <section class="padding-right: 200px;">
```

Comment:
using class attribute but content is for a style.

Reply:
This class does not work. It was deleted.